packages/patternfly-4/react-table/src/components/Table/utils/decorators/classNames.ts

```diff
@@ -13,6 +13,7 @@ export const Visibility = pickProperties(styles.modifiers, [
   'visibleOnXl'
 ]);
 
-export default (...classNames) => () => ({
+// tslint:disable-next-line:no-shadowed-variable
+export const classNames = (...classNames: any[]) => () => ({
```

Comment:
would `classNames` not be a `string[]`?

Reply:
seems Ok... i was thinking of the case classNames are passed as an object... but it think that's fine for now. Updating!